docs/operations/troubleshoot.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting kOps clusters
+
+The first step to debugging a kOps cluster is to run `kops validate cluster --name <clustername> --wait 10m`. If the cluster has not validated by then, something is wrong.
+
+# The Control Plane
+
+If the above-mentioned command complains about an unavailable API server, it means the control plane isn't working properly.
+In order to diagnose further, you need to log into one of the control plane nodes.
+
+Run `kops get instances` (1.19+) or look in the AWS console to identify a node with the master role. Then ssh into the IP address listed.
+
+The logs on the control plane resides in`/var/log`. Assume the logs are there unless otherwise noted.
+
+## Nodeup
+
+Nodeup is the process responsible for the initial provisioning of a node. It is a oneshot systemd service called `kops-configuration.service`. You can see the logs for this service running `journalctl -u kops-configuration.service`.
+
+If it succeed, you should be able to see the following log entries:
+
+```
+nodeup[X]: success
+systemd[1]: kops-configuration.service: Succeeded.
+systemd[1]: Finished Run kops bootstrap (nodeup).
```

Comment:
Would it be a good idea to add a similar entry for Protokube?

Reply:
There is a number of things we can add here. Some things around channels could be relevant wrt protokube. but perhaps easier to add as part of helping with a support question on slack. Most of the other content comes from that.

Reply:
OK